_posts/2023-01-11-install-and-manage-applications.md

```diff
@@ -0,0 +1,342 @@
+  - `apx init --apk` creates/reinitializes an Alpine container.
+
+- You can initialize the container graphically by clicking on the `+` icon in the Vanilla Control Center. (Vanilla Control Center also allows you to enter the apx container graphically.)
+
+- apx works by creating minimal containers from the distribution's docker image. And it tightly integrates with the host using a distrobox backend.
+
+### Installing Applications in Ubuntu Container
+
+- You can install applications in the Ubuntu container using the following command:-
+
+```bash
+apx install <packages>
+```
+
+- This command will automatically detect the desktop file entry in the package and add it to the application menu and Vanilla Control Center.
+
+- Alternatively, you can install the applications by entering the Ubuntu container using the following commands:-
+
+```bash
+apx enter
+sudo apt install <packages>
+exit
+apx export <packages>
+```
+
+### Uninstalling Applications from the Ubuntu Container
+
+- You can uninstall applications from the Ubuntu container using the following command:-
+
+```bash
+apx remove <packages>
+```
+
+- This command will automatically detect and remove the desktop file entry.
+
+- Alternatively, you can uninstall the applications by entering the Ubuntu container using the following commands:-
+
+```bash
+apx enter
+sudo apt remove <packages>
+exit
+apx unexport <packages>
+```
+
+### Installing Applications in the Fedora Container
+
+- You can install applications in the Fedora container using the following command:-
+
+```bash
+apx install --dnf <packages>
+```
+
+- This command will automatically detect the desktop file entry in the package and add it to the application menu and Vanilla Control Center.
+
+- Alternatively, you can install the applications by entering the Fedora container using the following commands:-
+
+```bash
+apx enter --dnf
+sudo dnf install <packages>
+exit
+apx export --dnf <packages>
+```
+
+### Uninstalling Applications from the Fedora Container
+
+- You can uninstall applications from the Fedora container using the following command:-
+
+```bash
+apx remove --dnf <package>
+```
+
+- This command will automatically detect and remove the desktop file entry.
+
+- Alternatively, you can uninstall the applications by entering the Fedora container using the following commands:-
+
+```bash
+apx enter --dnf
+sudo dnf remove <packages>
+exit
+apx unexport --dnf <packages>
+```
+
+### Installing Applications in the Arch Linux Container
+
+- You can install applications in the Arch Linux container using the following command:-
+
+```bash
+apx install --aur <packages>
+```
+
+- This command will automatically detect the desktop file entry in the package and add it to the application menu and Vanilla Control Center.
+
+- Alternatively, you can install the applications by entering the Arch Linux container using the following commands:-
+
+```bash
+apx enter --aur
+sudo pacman -S <packages>
```

Comment:
`pacman -S` doesn't refresh the repos. It's probably better to use `pacman -Syu`

Reply:
I used to use Arch BTW

Reply:
So, hmm `pacman -Syu <package>`?

Reply:
Yes. `-u` updates packages and `-y` refreshes the repos

Reply:
```suggestion
sudo pacman -Syu <packages>
```